### PR TITLE
Added Base Files  and Layers

### PR DESCRIPTION
**Pull request by MohamedMagdy-99**

### CODE REVIEW

File: AutomotiveBC_Team/MCAL/Interrupt Driver/ISR.c [hunk 1]
```diff
@@ -9,7 +9,7 @@
 
 /*- GLOBAL STATIC VARIABLES
 -------------------------------*/
-static volatile uint32_t gu32_OvfCounts = 0;
+static volatile uint32_t gu32_OvfCounts = Initial_Value;
```
Comment (AhmedNabil18) [inline AutomotiveBC_Team/MCAL/Interrupt Driver/ISR.c]: Why ???!!!!

**Comment by MohamedMagdy-99** [comment]
troll

File: AutomotiveBC_Team/MCAL/Interrupt Driver/ISR.c (hunk 1 above)
Comment (MohamedMagdy-99) [reply AutomotiveBC_Team/MCAL/Interrupt Driver/ISR.c]: troll bro

**Comment by MohamedMagdy-99** [comment]
gg

**Pull request merged**